### PR TITLE
feat: Add created/closed room prometheus metric

### PR DESCRIPTION
**Pull request by erikzenker**
This PR adds the metrics `palava_room_created_total` and `palava_room_closed_total` which we can use to calculate the number of open rooms

### CODE REVIEW

File: lib/rooms/room.ex [hunk 1]
```diff
@@ -125,6 +126,7 @@ defmodule SignalTower.Room do
         send_peer_left(next_members, peer_id)
         {:ok, {room_id, next_members}}
       else
+        SignalTower.PrometheusStats.room_closed()
```
Comment (farao) [inline lib/rooms/room.ex:129]: This must also work on l. 61, can you please check it again?

Comment (farao) [reply lib/rooms/room.ex:129]: I was wrong, leave is also (and actually most importantly) called in l.108 when the websocket process goes away, so this is the (only) correct spot to put this - sorry for the confusion.

File: lib/rooms/prometheus_stats.ex [hunk 2]
```diff
@@ -31,5 +42,6 @@ defmodule SignalTower.PrometheusHTTPHandler do
     {:ok, reply, :no_state}
   end
 
+  @spec terminate(any, any, any) :: :ok
```
Comment (farao) [inline lib/rooms/prometheus_stats.ex:45]: I guess if we use spec then we should do it everywhere (and then this should be a separate PR. Please remove this for now.

Comment (erikzenker) [reply lib/rooms/prometheus_stats.ex:45]: I wonder where this line is coming from. I guess the elixir vscode plugin sneaked it in. Will remove the line.

Comment (farao) [reply lib/rooms/prometheus_stats.ex:45]: yes...?

Comment (erikzenker) [reply lib/rooms/prometheus_stats.ex:45]: yes? what?

Comment (farao) [reply lib/rooms/prometheus_stats.ex:45]: Didn't you want to remove the line?

Comment (erikzenker) [reply lib/rooms/prometheus_stats.ex:45]: Sure

Comment (erikzenker) [reply lib/rooms/prometheus_stats.ex]: Fixed

File: lib/rooms/room.ex (hunk 1 above)
Comment (erikzenker) [reply lib/rooms/room.ex:129]: Np

**Comment by farao** [comment]
Please resolve the conflicts that I can merge this

**Comment by erikzenker** [comment]
@farao master is red. The merge #4 which lead to the current conflict with my branch introduced errors. These errors need to be fixed to get a green master again.

**Comment by erikzenker** [comment]
Here the link to CI https://github.com/farao/signaltower/runs/627738250?check_suite_focus=true

**Comment by farao** [comment]
> @farao master is red. The merge #4 which lead to the current conflict with my branch introduced errors. These errors need to be fixed to get a green master again.

Sorry, I fixed that. But it also says "This branch has conflicts that must be resolved", I think that's because Paul restructured some things in the meantime...

**Comment by erikzenker** [comment]
> This branch has conflicts that must be resolved

Yes, just wanted to rebase onto a valid master.

**Comment by erikzenker** [comment]
> I fixed that

Thx a lot !
